DirectEmployers/MyJobs
# [PD-3464] cleanup calendar components

## What's this PR do?
PR cleans up the calendar components

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of any analytics app or view the code

## How should this be manually tested?
Pull down the repository. Go into an analytics application and make sure the calendar is still working alone with viewing the code

## Any background context you want to provide?
